clinic/clinic-user-guide-for-tiup.md

```
@@ -85,19 +85,19 @@ PingCAP Clinicを使用する前に、Diag（ PingCAP Clinicが提供するデ
         tiup diag config clinic.token ${token-value}
         ```
 
-3.  Diag に`region`設定します。
+3.  Diag に`region`を設定します。
 
     `region` 、データの圧縮に使用する暗号化証明書と、データのアップロード時に使用する対象サービスを決定します。例:
 
     > **Note:**
     >
     > -   Diag v0.9.0 以降のバージョンでは設定`region`サポートされます。
```

markdownlint-cli2 (0.23.2)

[warning] Spaces after list markers
Expected: 1; Actual: 3

(MD030, list-marker-space)